api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -147,6 +150,10 @@ private void expireActiveClaim(Claim claim, List<LocalDate> dateOfBirthOfChildre
         claimMessageSender.sendReportClaimMessage(claim, dateOfBirthOfChildren, UPDATED_FROM_ACTIVE_TO_EXPIRED);
     }
 
+    private void setCardStatusToPendingCancellation(Claim claim) {
+        claim.updateCardStatus(PENDING_CANCELLATION);
```

Comment:
You should call `claimRepository.save(claim)` here.

Reply:
Done.

Reply:
Should we be looking to save the claim once rather than in two separate places?

Reply:
Possibly, but that might be a bit more tricky than is practicable as not all the routes through this code call save.